### PR TITLE
added "KCL: Restart Language Server" command to restart the KCL language server

### LINKED ISSUE #38: [Enhancement] Add the command to restart the KCL language server
Taking VS Code plugin commands of Bazel and CodeQL as an example

image

### PR DESCRIPTION
**Pull request by utnim2**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [✅ ] Y 

Fix #38 

#### 2. What is the scope of this PR (e.g. component or file name):
src/extension.ts
package.json

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [✅ ] Affects user behaviors

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ✅ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

No

### CODE REVIEW

**Comment by utnim2** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/extension.ts [hunk 1]
```diff
@@ -28,7 +29,11 @@ export async function activate(context: vscode.ExtensionContext) {
 	// 	}
 	// });
 
+	const restartLanguageServerCommand = vscode.commands.registerCommand('kcl.restartLanguageServer', restartLanguageServer, "KCL: Restart Language Server");
+	console.log('KCL: Restart Language Server command registered');
+
 	context.subscriptions.push(autoCompletionProvider);
+    context.subscriptions.push(restartLanguageServerCommand);
```
Comment (Peefy) [inline src/extension.ts:36]: Format this code.

**Comment by Peefy** [comment]
I've added some comments. @He1pa Can you help review it?

**Comment by utnim2** [comment]
> I've added some comments. @He1pa Can you help review it?

Any updates @Peefy apart from the formating part

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**